cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,115 @@
+# cloudcomposerdiff
+
+Hey there ! If you need to diff two Cloud Composer environments, this tool is for you.
+
+cloudcomposerdiff is a python command line tool to diff two Cloud Composer environments.
+
+![gif showing environments with some matches & some differences](img/some_matches.gif)
+
+As per the example above, this tool looks at various Cloud Composer environment
+attributes and diffs them. It shows which attributes have different values across
+environemnts, and what those values are. It also shows which attributes have a common
+value across environments, and what this common value is.
+
+The tool can compare the follow attributes of Cloud Composer environments:
+
+* Cloud Compser image version
+* AirFlow configuration overrides
+* PyPi packages
+* Environment variables
+
+When comparing two Cloud Composer environments that share no common attribute values
+the "matching_value" column will be empty as per the example below:
+
+![gif showing environments with zero matches](img/no_matches.gif)
+
+When comparing two Cloud Composer environments that share common attributes values
+the "matching_value" column will be populated and the "env_1_value" and "env_2_value"
+columns will be empty. This is shown in the example below.
+
+![gif showing environments with lots of matches](img/lots_of_matches.gif)
+
+
+## Running the application
+
+### Prerequisites
+
+ * Python version: application has been tested on python version 3.10
+ * Composer version:  compatible with both Composer 1 and Composer 2
+ * Composer version:  compatible with all Composer configurations
+ * GCP IAM Permissions: executing user needs composer.environments.get [permission](https://cloud.google.com/composer/docs/how-to/access-control#permissions_for_api_methods)
+
+### Clone repository
+
+[Clone the code](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) from GitHub to your local machine
+
+### Install locally with pip
+
+navigate to the cloudcomposerdiff directory
+
+from within the cloudcomposerdiff directory run the following command
+
+```shell
+pip install cloudcomposerdiff
```

Comment:
Can you double check this command? I am not able to run it as is because right now this tells pip to look for a package in PyPI named `cloudcomposerdiff`

Reply:
I think it needs to be `pip install .` or `pip install -r requirements.txt`

Reply:
(I think it's `pip install .` because that looks for our setup.py)

Reply:
cc @chalmerlowe for his wisdom here 😁

Reply:
Typically, if you are in the directory and want to do a local `pip install`, then it is customary to run `pip install .`.

Since the comment above indicates that this is intended to run the application (instead of installing in editable mode for development purposes) I would expect `pip install .` to be the appropriate answer.

Reply:
thank you @chalmerlowe !!

Reply:
Thank @leahecole and @chalmerlowe for the feedback & direction here! 

I spun up a new VM, clone the repo and ran into the exact same issue you did. 

I've updated the README with this instruction.